extensions/amp-story-auto-ads/0.1/story-ad-ui.js

```diff
@@ -215,6 +216,12 @@ function createPageOutlink_(doc, uiMetadata, container) {
     }
   }
 
+  Services.extensionsFor(getWin(doc)).installExtensionForDoc(
```

Comment:
Services is not defined :)

Reply:
Forgot an "import Services" somewhere, will add it